vaadinfrontend/src/main/java/life/qbic/datamanager/views/notifications/StyledNotification.java

```diff
@@ -10,21 +9,18 @@
  *
  * @since <version tag>
  */
-public class StyledNotification {
+public class StyledNotification extends Notification {
```

Comment:
Because I needed to add listeners to the Notification and the old one swallowed the respective methods provided by Notification. Listeners are needed because the Notification popping up and vanishing breaks the UI of the Spreadsheet, which needs to be refreshed.